assignment1/problem2.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <script>
+        function doArithmeticOperation(operationCode) {
+            let number1 = document.getElementById('number1').value;
+            let number2 = document.getElementById('number2').value;
+
+            if (number1 == '') {
+                alert('Please enter the both number1.');
+                return false;
+            }
+            if (number2 == '') {
+                alert('Please enter the both number2.');
+                return false;
+            }
+
+            if (isNaN(parseInt(number2))) {
+                alert('Input is invalid');
+                return false;
+            }
+            if (isNaN(parseInt(number1))) {
+                alert('Input is invalid');
+                return false;
+            }
+
+            if (operationCode == 1) {
+                document.write('Addition is : ${(parseInt(number1) - parseInt(number2))}');
+            } else {
+                document.write('Subtraction is : ${(parseInt(number1) - parseInt(number2))}');
+            }
+        }
+    </script>
+</head>
+<body>
+<center>
+    <form name="myForm">
+        <input type="text" id="number1"><br><br>
+        <input type="text" id="number2"><br><br>
+        <button onclick="doArithmeticOperation('1')">Add</button>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
what is `1` here, Instead declare constants and use them.

```
const OPERATIONS = {
  ADD: 'ADD',
  SUBSTRACT: 'SUBSTRACT', 
}
```

`doArithmeticOperation(OPERATIONS.ADD);`